### DISCUSSION TITLE
False Positive?

### DISCUSSION THREAD

**Discussion by NahimGFX** [baffler/Transparent-Twitch-Chat-Overlay#252] [Q&A]
Analyzing the latest version in virus total it comes out as a threat, I don't know if it is a false positive, also when installing it Windows tells me not to run it

**Comment by IAmAProfessionalGamer** [comment]
It's a false positive. I run Malwarebytes and it hasn't reported anything. Windows can be ignored, you know where the code comes from and it's just being a nag.
